flask.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+SCRIPT_PATH=`dirname "$0"`
+export INNOMETRICS_BACKEND_PATH="${SCRIPT_PATH}/"
```

Comment:
This script may be executed from other folder and environment variable will wrong in that case